### PR TITLE
created email template for sign-up

### PR DESCRIPTION
**Pull request by KornelijusGlinskas**
@dominykasmurauskas You've asked an email template so you could kick-off the emailing system.  I'll edit it in the future but for now, you can use it as a template.

It looks like this - https://ibb.co/Wfxqqvx

### CODE REVIEW

**Comment by dominykasmurauskas** [comment]
before merging, translations must be added & template should be used for 3 different mail types.
I will get on this if I have time. :)

**Comment by dominykasmurauskas** [comment]
it can now be merged to master for smoother connection to the mailing system, because this template is not active yet. it will be tested & updated on development environment until it's fully prepared.

File: src/Controller/RegistrationController.php [hunk 1]
```diff
@@ -40,11 +48,14 @@ public function register(
                     $form->get('plainPassword')->getData()
                 )
             );
+            $user->setVerificationKey($user->generateVerificationKey());
```
Comment (tadas-nfq) [inline src/Controller/RegistrationController.php]: Is there a case where you only need to generate key and not set it to user?
If not, maybe you should automatically set it to verificationKey in User->generateVerificationKey method.

File: src/Controller/RegistrationController.php [hunk 2]
```diff
@@ -40,11 +48,14 @@ public function register(
...
 
             $entityManager = $this->getDoctrine()->getManager();
             $entityManager->persist($user);
             $entityManager->flush();
 
+            $this->sendSignupEmail($user, $translator, $mailer);
+
             // do anything else you need here, like send an email
```
Comment (tadas-nfq) [inline src/Controller/RegistrationController.php]: Remove this comment.

File: templates/emails/client-verified.html.twig [hunk 3]
```diff
@@ -1,7 +1,35 @@
-<h3>{{ "mailer.title.verification"|trans }}</h3>
+{% extends '/emails/template.html.twig' %}
 
-{{ "mailer.greeting"|trans({'%user%': user}, "messages") }}
+{% block heading %}
+    <div
+            style="font-family:Poppins, Helvetica;font-size:20px;font-weight:bold;line-height:1;text-align:left;color:#1A1A1A;"
```
Comment (tadas-nfq) [inline templates/emails/client-verified.html.twig]: I think element with parameters should be in one line. Like <div parameters***>

File: templates/emails/client-verified.html.twig [hunk 4]
```diff
@@ -1,7 +1,35 @@
...
 
-{{  "mailer.verification.success"|trans }}
-{{ "mailer.cancellation.cancel"|trans }}
-<a href="{{ absolute_url(path('client_cancel', {verificationKey: key})) }}">{{ "mailer.cancellation.button"|trans }}</a>
+
+{% endblock %}
+
+
+{% block button %}
+    <td
```
Comment (tadas-nfq) [inline templates/emails/client-verified.html.twig]: And if line is too long you can press Ctrl + Alt + L to automatically format long lines.

**Pull request merged**